applications/smart-distancing/eval/README.md

```diff
@@ -0,0 +1,52 @@
+# Smart Social Distancing - Performance Metrics for Object Detection
+
+This module provides easy-to-use functions which enables you to evaluate available exported results from detectors.
+
+### How to use
+**Step 1: Build Image**
+```
+cd neuralet/applications/smart-distancing/eval
+# Build Docker image
+docker build -f Dockerfile-eval -t detector-eval .
+```
+**Step 2: Create The Input Files**
+
+* Create a separate ground truth text file for each image in the directory `neuralet/applications/smart-distancing/eval/eval_files/groundtruths/`.
```

Comment:
mention that this path is default and user can use any other path.

Reply:
It's mentioned on the table.